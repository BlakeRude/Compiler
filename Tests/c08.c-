// C-S21
bool x[3];
bool y[4];

main()
{
    x[2] = true;
    y[1] = false;
    outputb(x[2]  or  y[1]);
    outnl();
}
