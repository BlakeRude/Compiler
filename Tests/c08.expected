Number of warnings: 0
Number of errors: 0
------------------------------------
Loading file: c08.tm
T
Bye.
